### ISSUE TITLE
Bug: Popover examples with no focusable content do not move focus to popover surface

### ISSUE THREAD

**Issue by smhigley** [microsoft/fluentui#28106]
We should expose the content of the popover in some way to screen reader users. If a popover were functioning as a tooltip, we could use something like `aria-describedby` without moving focus, but for all the examples in the docs page, it looks like we should be moving focus to the PopoverSurface, which we can do by adding `tabindex="-1"`.

Creating an issue for myself to do this, and add authoring docs.

**Comment by smhigley** [comment]
Resolved in #29556
